word_language_model/model.py

```diff
@@ -42,7 +42,7 @@ def __init__(self, rnn_type, ntoken, ninp, nhid, nlayers, dropout=0.5, tie_weigh
     def init_weights(self):
         initrange = 0.1
         nn.init.uniform_(self.encoder.weight, -initrange, initrange)
-        nn.init.zeros_(self.decoder)
+        nn.init.zeros_(self.decoder.weight)
```

Comment:
Is there a point to invoke `nn.init.zeros_` here?
The next line re-initializes `self.decoder.weight`.